_posts/luznette/2020-09-16-luznette-functional-turtle-exercise .md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+author: luznette
+title: "Janette's Functional Turtle Exercise"
+---
+
+This assignment was tricky only because I had to find a previous turtle I made that had repetitive code, but also didn't change it all the much. They both still do the same thing, just broke it down into three parts and into three ```def```, and then calling the statements at the end. Hopefully, this would make it easier if I were to write longer code and need to call the circles, the background, or the message.
```

Comment:
Nice work using markdown to refer to `def`! The triple backticks you've used here will set it off in its own little block. To have code show up `inline like this`, use single backticks.

No need to revise unless it bothers you when you see this post on the site!